concepticondata/conceptlists.tsv

```diff
@@ -429,4 +429,5 @@ Capell-1956-558	Capell, Arthur	1956		558	basic	English	North Australian language
 Healey-1970-364	Healey, Alan	1970		364	basic	English	Awyu-Dumut Languages		Healey1970	Healey1970	This list contains concepts that are reconstructed for Awyu-Dumut languages.	1049-1058 
 Auderset-2023-240	Auderset, Sandra,  and Greenhill, Simon J., and DiCanio, Christian T., and Campbell, Eric W.	2023		209	basic	English	Mixtecan Languages		Auderset2023	Auderset2023	This is a basic wordlist collected for identifying the relationships between Mixtecan languages. Note that the paper talks about a list of 209 items. However, as our comparison of the original data and the phylogenetic analysis showed, the correct number of items used was 240. For this reason, we list the list in the form which we consider as original. We also dropped the Spanish gloss originally listed, since it contained certain parts that were not translated.		
 Valenzuela-2023-44	Valenzuela, Pilar and Zariquiey, Roberto	2023		44		English	Panoan and Tacanan languages	https://doi.org/10.20396/liames.v23i00.8670150	Valenzuela2023		This list is derived from a study that investigates the genetic relationship between the Panoan and Tacanan families. It compares Proto-Pano and Proto-Takana reconstructions proposing lexical and grammatical cognates in support of the Pano-Takana hypothesis.	1-53	
-Girard-1971-559	Girard, Victor	1971		559		Spanish	Tacanan languages		Girard1971		This list is extracted from a study for the classification of the Panoan and Tacanan families. The study provides cognates for Tacanan phonology, morphology, and lexicon. It also presents a synchronic description and classification of Tacanan languages.	50-138	
\ No newline at end of file
+Girard-1971-559	Girard, Victor	1971		559		Spanish	Tacanan languages		Girard1971		This list is extracted from a study for the classification of the Panoan and Tacanan families. The study provides cognates for Tacanan phonology, morphology, and lexicon. It also presents a synchronic description and classification of Tacanan languages.	50-138	
+Seifart-2015-414	Seifart, Frank and Echeverri, Juán Álvaro	2015		414		English	Bora and Muinane		Seifart2015		This list contains concepts used for a reconstruction of the proto Bora-Muinane phonological system and of 415 word forms. It also reassess the genealogical relationship of Bora-Muinane with Witoto, Ocaina, and Nonuya.	279-311	
```

Comment:
It is "Juan", not Juán"

Reply:
"It also reassess the genealogical relationship of Bora-Muinane with Witoto, Ocaina, and Nonuya"

No, it does not.